seed/csharp-model/examples/src/SeedExamples/Types/Metadata.cs

```diff
@@ -189,9 +189,9 @@ JsonSerializerOptions options
 
             var value = discriminator switch
             {
-                "html" => json.GetProperty("value").Deserialize<string>(options)
+                "html" => json.GetProperty("value").Deserialize<string?>(options)
                 ?? throw new JsonException("Failed to deserialize string"),
-                "markdown" => json.GetProperty("value").Deserialize<string>(options)
+                "markdown" => json.GetProperty("value").Deserialize<string?>(options)
```

Comment:
Is `string?` here a string because `value` is a nullable property? If so, we should remove the `?? throw ...`.
Otherwise, good as is

Reply:
It's not actually a nullable property -- the generator won't put a throw on a nullable property deserializing as null. This is intentional